data-prepper-plugins/sqs-source/src/main/java/org/opensearch/dataprepper/plugins/source/sqssource/config/QueuesOptions.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source.sqssource.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Duration;
+import java.util.List;
+
+public class QueuesOptions {
+
+    private List<String> urls;
+
+    @JsonProperty("polling_frequency")
+    private Duration pollingFrequency;
+
+    @JsonProperty("batch_size")
+    private Integer batchSize;
+
+    @JsonProperty("number_of_threads")
+    private Integer numberOfThreads = 1;
```

Comment:
Do these need validation annotations?

Reply:
Thanks @graytaylor0 for your review, Please suggested as per the current implementation we are not using any validation and using some default value incase user not provided.

Reply:
Having a default doesn’t omit the need for validations. What would happen if the user set number of threads to a negative number?